### ISSUE TITLE
[Question/Bug] Does unlocking all skins with TEKLauncher unlock 90% explorer notes achievements in Steam?

### ISSUE THREAD

**Issue by vavavr00m** [Nuclearistt/TEKLauncher#59]
TL;DR: 90% Explorer Notes-related Steam achievements and skins were unlocked in a non-dedi session even after resetting the launcher settings, removing mods & DLCs installed by the launcher and verifying game files twice.

I have the base and all the DLCs purchased. Last time I played seriously was almost a year ago as seen in the screenshot. I did join some servers here and there this year but didn't last more than 3 days. I started writing a script over a year ago to automate fetching all the spawncodes with the use of readily available spawncode generators but shelved it because I didn't have the time to learn and write batch or PS. Out of the blue, there's a renewed interest in continuing the script. I redownloaded all the stuff, some (a few mods and some DLCs) with the use of this launcher because the official Steam client was downloading insanely slow for me no matter which server was chosen. I saw the unlock skins and fix bloom options and applied both. Eventually I decided to stop using the TEKLauncher; we were trying to figure out an issue with the spawncode generators, so, removed the mods & DLCs installed with it, reset the launcher settings, redownloaded with the official client and verified Steam game files twice. Today I ran a fresh non-dedi session to test something and 90% of the explorer notes achievements were suddenly unlocked as well as the skins. I did enter "cheat" in the console at some point and spawned some items such as Wyvern Milk, Nameless Venom, and Blood Pack but nothing I know of that could've cause to get all the skins and 90% explorer notes achievements. I'm certain I got the achievements & skins even before entering the cheat commands. All I fiddled with recently were TEKLauncher and the spawncode generators - I haven't even joined a dedi server yet after reinstalling all the game files nor have played solo. I asked a friend, who doesn't know about TEKLauncher but who was almost always with me whenever I join a new server, to try running a non-dedi session but there were no new achievements nor new skins. I have never set foot in Valguerro, Scorched Earth, and The Center, have barely explored Ragnarok & Genesis pt1 lands, have gone to the deep seas of only Ragnarok, Crystal Isles, and Lost Island and certain to have not completed Fjörður, thus, it's highly unlikely that I have gotten 90%. If it matters, usually played modded (not sure if achievements will be unlocked on modded because some games don't allow achievements to be unlocked if the server is modded) and have only played on the official for a day, twice with non-modded servers with adjusted rates but didn't last long enough for me to find a siginificant number of notes. If TEKLauncher's option to unlock skins really was what had caused it, I wish there's gonna be a warning before applying it because some people, like me, prefer unlocking Steam achievements on their own. 

![image](https://github.com/Nuclearistt/TEKLauncher/assets/4675132/38c0b6b3-f8af-4ccd-b25e-10bdc48ddce1)

**Comment by Nuclearistt** [comment]
The unlocked status for most skins is determined by ShooterGame\Saved\LocalProfiles\PlayerLocalData.arkprofile file in your game directory. If you want to reset those skins, just delete that file, but keep in mind that it's also responsible for unlocked haircuts, maps' fog of war, map waypoints and "unlocked" explorer notes (making them readable from menu even if they weren't discovered in current session), so all that data will be lost too
